Traceback (most recent call last):
  File "test/verify/check-setroubleshoot", line *, in testTroubleshootAlerts
    b.wait_in_text("body", "No SELinux alerts.")
*
Error: timeout
